tests/fixtures/issues/issue-635.php

```diff
@@ -48852,7 +48852,7 @@
             'isFake' => false,
             'isAnonymized' => false,
             'isModified' => false,
-            'CssVersion' => '0',
```

Comment:
All of the baidu box app useragents have `CssVersion` as `3` now since they're all a pretty recent WebKit/Blink fork, so just updated these tests to match that.